integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsRestart.java

```diff
@@ -231,26 +233,7 @@ public void testServerPodsRestartByChangingZImage() throws Exception {
 
       if (BaseTest.SHARED_CLUSTER) {
```

Comment:
This check can be removed and you can simply use 
getWeblogicImageName() + ":" + getWeblogicImageDevTag()

The getWeblogicImageDevTag() method and a field can be added in BaseTest.java